lib/kitchen_sink/integer.ex

```diff
@@ -0,0 +1,16 @@
+defmodule KitchenSink.Integer do
+  @moduledoc """
+  this module is for Integer functions
+  """
+
+  @doc """
+  Make sure that we get an integer either from an integer or string
+
+  iex> ensure_integer("1")
+  1
+  iex> ensure_integer(2)
+  2
+  """
+  def ensure_integer(i) when is_integer(i), do: i
```

Comment:
`Integer.ensure_integer/1` might be a bit much. 

Naming is hard; some raw thoughts would be naming the module `Convert` or `Cast` and the function `to_integer` or something along those lines?